test/connector/test_milvus_stager.py

```diff
@@ -0,0 +1,164 @@
+import pytest
+
+from unstructured_ingest.v2.processes.connectors.milvus import (
+    MilvusUploadStager,
+    MilvusUploadStagerConfig,
+)
+
+
+@pytest.mark.parametrize(
+    ("given_element", "given_field_include_list", "then_element", "then_error"),
+    [
+        ({"x": "y"}, None, {"x": "y", "is_continuation": False}, False),
+        (
+            {
+                "type": "UncategorizedText",
+                "element_id": "be34cd2d71310ec72bfef3d1be2b2b36",
+            },
+            ["type", "element_id"],
+            {
+                "type": "UncategorizedText",
+                "element_id": "be34cd2d71310ec72bfef3d1be2b2b36",
+            },
+            False,
+        ),
+        (
+            {
+                "type": "UncategorizedText",
+                "element_id": "be34cd2d71310ec72bfef3d1be2b2b36",
+                "parent_id": "qwerty",
+            },
+            ["type", "element_id"],
+            {
+                "type": "UncategorizedText",
+                "element_id": "be34cd2d71310ec72bfef3d1be2b2b36",
+            },
+            False,
+        ),
+        (
+            {
+                "type": "UncategorizedText",
+                "parent_id": "qwerty",
+            },
+            ["type", "element_id"],
+            {},
+            True,
+        ),
+    ],
+)
+def test_milvus_stager_processes_elements_correctly(
+    given_element, given_field_include_list, then_element, then_error
+):
+    config = MilvusUploadStagerConfig(fields_to_include=given_field_include_list)
+    stager = MilvusUploadStager(upload_stager_config=config)
+
+    try:
```

Comment:
If I understand correctly you are expecting KeyError to happen here when given then_error data. 
You could consider this approach:
` if then_error:
        with pytest.raises(KeyError):
            stager.conform_dict(data=given_element)`
  it simplifies the logic plus would generate better infor for the user if somethine else goes wrong here.

Reply:
I have added two test branches for true and false cases of then_error argument.